src/main/java/com/todolist/app/springmvcboottodolist/models/Status.java

```diff
@@ -0,0 +1,29 @@
+package com.todolist.app.springmvcboottodolist.models;
+
+/**
+ * These are statuses, which user can mark his tasks
+ */
+public enum Status {
+    /**
+     * This status means that the task is done
+     */
+    COMPLETED,
+    /**
+     * This status means that the task hasn't been done yet
+     */
+    INCOMPLETED,
+    /**
+     * This status means that the task is being worked
+     */
+    INPROGRESS,
+    /**
+     * This status means that the task is in a queue
+     */
+    PENDING;
+
+    @Override
+    public String toString() {
```

Comment:
Why do we need this method?

Reply:
This method will be used in future on another jsp page for printing Status